src/cargo/core/compiler/output_depinfo.rs

```diff
@@ -13,6 +13,12 @@ use log::debug;
 
 /// Bacially just normalizes a given path and converts it to a string.
 fn render_filename<P: AsRef<Path>>(path: P, basedir: Option<&str>) -> CargoResult<String> {
+    fn wrap_path(path: &Path) -> CargoResult<String> {
```

Comment:
Whats gained by doing this?  Personally, I avoid inner functions unless I'm doing it for generics.

Reply:
One less item to show in the list of functions in rustdoc? But you're right not many gains.